MPCAutofill/cardpicker/templates/cardpicker/legal.html

```diff
@@ -1,48 +1,80 @@
 {% extends 'cardpicker/base.html' %}
 
-{% block title %} MPC Autofill Privacy Policy {% endblock %}
+{% block title %} {{ SITE_NAME }} Legal {% endblock %}
 
 {% block content %}
+    
+    
+    <h2>About {{ SITE_NAME }}</h2>
+
+    <p>
+        {{ SITE_NAME }} is built using an open source django project primarily developed

```

Comment:
Added! I also adjusted the following wording so MPC Autofill is named explicitly rather than SITE NAME and then explained that MPC Autofill "and any forks" are open source, you can review the new wording soon as I commit :)